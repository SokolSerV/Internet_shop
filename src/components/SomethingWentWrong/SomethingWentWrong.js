export default function SomethingWentWrong() {
    return (
        <div style={{ textAlign: "center" }}>
            <h2>Something went wrong ...</h2>
            <h3>Try to reload page</h3>
            <button onClick={() => window?.location?.reload()}>Reload</button>
        </div>
    );
}
