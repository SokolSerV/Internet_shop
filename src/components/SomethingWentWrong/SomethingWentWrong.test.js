import { render } from "@testing-library/react";

import SomethingWentWrong from "./SomethingWentWrong";

describe("SomethingWentWrong component Smoke test (Snapshot test)", () => {
    test("should SomethingWentWrong render", () => {
        const { asFragment } = render(<SomethingWentWrong />);

        expect(asFragment()).toMatchSnapshot();
    });
});
