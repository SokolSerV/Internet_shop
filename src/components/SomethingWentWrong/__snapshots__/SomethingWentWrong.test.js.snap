// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SomethingWentWrong component Smoke test (Snapshot test) should SomethingWentWrong render 1`] = `
<DocumentFragment>
  <div
    style="text-align: center;"
  >
    <h2>
      Something went wrong ...
    </h2>
    <h3>
      Try to reload page
    </h3>
    <button>
      Reload
    </button>
  </div>
</DocumentFragment>
`;
